.github/workflows/PR.yml

```diff
@@ -43,61 +43,42 @@ jobs:
                  fi
                done < files.txt
 
-   api_check:
+   validate-abi:
```

Comment:
validate-api ?

Reply:
Took that from master. API is probably correct. Will change.

Reply:
I changed the name to be `validate-abi` becuase it's more accurate. The BCV dumps have a `.api` extension, but this might be changed to `.abi`

https://github.com/Kotlin/binary-compatibility-validator/issues/198

However, if `validate-api` is more understandable, then that is more important 👍

Reply:
Good point! The BCV's README says
> binary API of a JVM part of a Kotlin library

So we have it all. (Why am I thinking of "Cake by the Ocean" now?)

What if we'd follow their naming and go with API for now, then switch to ABI when they do? That way, no one needs to translate names.

Reply:
Sounds good to me!